examples/all-clusters-app/esp32/main/Button.cpp

```diff
@@ -28,51 +27,47 @@
 #include "esp_log.h"
 #include "esp_system.h"
 
+#include "AppTask.h"
 #include "Button.h"
+#include "Globals.h"
+#include "ScreenManager.h"
 #include <lib/support/CodeUtils.h>
 #include <platform/CHIPDeviceLayer.h>
 
-extern const char * TAG;
 
-esp_err_t Button::Init(gpio_num_t gpioNum, uint16_t debouncePeriod)
-{
-    mGPIONum         = gpioNum;
-    mDebouncePeriod  = debouncePeriod / portTICK_PERIOD_MS;
-    mState           = false;
-    mLastPolledState = false;
+static const char * TAG = "Button.cpp";
 
-    esp_err_t err = gpio_set_direction(gpioNum, GPIO_MODE_INPUT);
-    if (err == ESP_OK)
-    {
-        Poll();
-    }
-    return err;
+static void IRAM_ATTR button_isr_handler(void * arg)
+{
+    uint32_t gpio_num = (uint32_t) arg;
```

Comment:
Previously there was debouncing mechanism in the file and it's removed. Have we confirmed the M5 stack GPIO does not require debouncing?